### ISSUE TITLE
Refactor Emissions Filter Logic Endpoint- PI-7 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2746]
## Code that needs to be refactored 

Scope of this ticket includes: 
1. Add emissions filter logic (/emissions-management/apportioned/attributes/applicable) endpoint to the emissions API 
2. Remove source category from emissions filter logic endpoint

**Comment by JanellC** [comment]
@tlageman-epa 
We decided to split the scope of this ticket.   As a apart of the scope of this ticket we will add the emissions filter logic endpoint to the emissions API and remove source category from the endpoint. Since the scope of this ticket was decreased, I updated the estimate from a 3 to a 2. 

In ZH ticket Update custom data download to use Emissions filter logic endpoint #2832, we will update the cdd to use the new emissions  filter logic endpoint and preform regression testing. Let me know if you have any questions.
